File: SA.Web/src/poco/UserSimpleDto.ts
import { RecordTableDto } from '@/poco';
export default interface UserSimpleDto {
    id: number;
    userName: string;
    isActive: boolean;
    isAgreementToTerms: boolean;
    sendingNews: boolean;
    customerId: number;
    language: string;
    created: Date;
    isFeePayed: boolean;
    email: string;
    phoneNumber: string;
    companyNumber: string;
    birthNumber: string;
    actualAuctions: RecordTableDto[];
    endedAuctions: RecordTableDto[];
}

File: SA.Application/Account/UserSimpleDto.cs
using System;

namespace SA.Application.Account
{
    public class UserSimpleDto
    {
        public virtual int Id { get; set; }

        public virtual DateTime? Created { get; set; }

        public virtual string UserName { get; set; }

        public virtual bool IsActive { get; set; }

        public virtual bool IsAgreementToTerms { get; set; }

        public virtual bool SendingNews { get; set; }

        public virtual string Token { get; set; }

        public virtual string Language { get; set; }

        public virtual int CustomerId { get; set; }

        public virtual bool IsFeePayed { get; set; }

        public virtual string Email { get; set; }

        public virtual string PhoneNumber { get; set; }
    }
}
